README.md

```diff
@@ -129,6 +119,11 @@ Follow our [Guide for Apache](./doc/example-config-apache.md) on how to run Spre
 
 Follow our [Guide for Nginx](./doc/example-config-nginx.md) on how to run Spreed WebRTC on the same host as Nextcloud in a Nginx subpath.
 
+## Running Spreed WebRTC and Nextcloud on different origins
+
+Running Spreed WebRTC and Nextcloud on different origins is not recommended, as you will not be able to use the Screensharing feature due to browser restrictions.
+If you still prefer to run them on different origins, please read the [README.md of v0.1.6](https://github.com/strukturag/nextcloud-spreedme/blob/owncloud-spreedme_v016/README.md#preparations).
```

Comment:
Well, that was the last version where README.md had a tutorial on how to use different origins.
I can however create a new document in doc/ on how to do so.